xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-api/src/main/resources/liveDataConfiguration.json

```diff
@@ -14,28 +14,33 @@
     "filters": [
       {
         "id": "text",
-        "operators": ["contains", "startsWith", "equals"]
+        "operators": ["contains", "startsWith", "equals"],
+        "defaultOperator": "contains"
       },
       {
         "id": "number",
         "operators": [
           {"id": "equals", "name": "="},
           {"id": "less", "name": "<"},
           {"id": "greater", "name": ">"}
-        ]
+        ],
+        "defaultOperator": "equals"
       },
       {
         "id": "boolean",
-        "operators": ["equals"]
+        "operators": ["equals"],
+        "defaultOperator": "equals"
       },
       {
         "id": "date",
         "operators": ["between", "before", "after", "contains"],
-        "dateFormat": "yyyy/MM/dd HH:mm"
+        "dateFormat": "yyyy/MM/dd HH:mm",
+        "defaultOperator": "between"
       },
       {
         "id": "list",
-        "operators": ["equals", "startsWith", "contains", "empty"]
+        "operators": ["equals", "startsWith", "contains", "empty"],
+        "defaultOperator": "contains"
```

Comment:
Note that contrarily to the other, it's not the first operator, but it matches the old behaviour of always using `contains` in the FilterList code. Changing the default to equals would be a breaking change.

Reply:
I agree, now problem is that JSON doesn't allow to put comments...